src/main/java/Task.java

```diff
@@ -0,0 +1,15 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
```

Comment:
function name: Status is still too ambiguous. The icon is specifically refers to if  the task is done or not.